aten/src/ATen/cuda/CUDAGraph.cpp

```diff
@@ -4,6 +4,7 @@
 #include <ATen/Functions.h>
 #include <c10/cuda/CUDACachingAllocator.h>
 #include <c10/cuda/CUDAFunctions.h>
+#include <torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp>
```

Comment:
Another issue shows up.  Some internal Windows jobs are failing with the following error:

```
FAILED because xplat\caffe2\aten\src\ATen\cuda\CUDAGraph.cpp: included an untracked header: xplat\caffe2\torch\csrc\distributed\c10d\ProcessGroupNCCL.hpp
```

Reply:
The diff above should fix that I guess?

Reply:
Well, the diff D49798453 fixes the first error when `ProcessGroupNCCL.hpp` isn't found.  Now, all the Linux builds are working, but the Windows builds are complaining about `ProcessGroupNCCL.hpp` is there but not tracked.  I think the new Windows build failure could be fixed here by adding `#ifdef USE_C10D_NCCL` or `#ifndef _WIN32` (NCCL is not available on Windows).  Then it would be good to go. @eqy Could you help update the PR for that?